### PR TITLE
az sp create 1

### PR DESCRIPTION
**Pull request by mkyc**
Fixed up to line `pass, err := az.GeneratePassword(32, 10)`

### CODE REVIEW

File: pkg/az/az_test.go [hunk 1]
```diff
@@ -77,7 +71,7 @@ func cleanupTestServicePrincipal(spObjectID, appObjectID string, t *testing.T) {
 // func TestShouldSuccessfullyCreateServicePrincipal(t *testing.T) {
 
 // 	// when
-// 	pass := GenerateServicePrincipalPassword()
+// 	pass := GeneratePassword()
```
Comment (erzetpe) [inline pkg/az/az_test.go:75]: You can uncomment those tests. It was done only for avoiding issue with SP.

File: pkg/az/ops.go [hunk 2]
```diff
@@ -43,27 +43,20 @@ func errFailedToGetEnvironment(err error) {
 		Msg("Failed to get environment.")
 }
 
-func errFailedToGetAuthrorizerFromCli(err error) {
+func errFailedToGetAuthorizerFromCli(err error) {
```
Comment (erzetpe) [inline pkg/az/ops.go:46]: This will be removed?

File: pkg/az/ops.go [hunk 3]
```diff
@@ -43,27 +43,20 @@ func errFailedToGetEnvironment(err error) {
...
 	logger.
 		Panic().
 		Err(err).
 		Msg("Failed to get Authrorizer from CLI.")
 }
 
-func errFailedToGetGraphAuthrorizer(err error) {
+func errFailedToGetGraphAuthorizer(err error) {
```
Comment (erzetpe) [inline pkg/az/ops.go:53]: This will be removed?

**Review by erzetpe — CHANGES_REQUESTED** [review]

File: pkg/az/ops.go (hunk 2 above)
Comment (mkyc) [reply pkg/az/ops.go:46]: Nope it's just typo fixed in: Auth**r**orizer

File: pkg/az/az_test.go (hunk 1 above)
Comment (mkyc) [reply pkg/az/az_test.go:75]: I will do it when I reach this part of tests.

File: pkg/az/ops.go (hunk 3 above)
Comment (mkyc) [reply pkg/az/ops.go:53]: Same here, just fixing typo.

**Pull request merged**